src/donejs-carousel-plugin.less

```diff
@@ -11,7 +11,7 @@ tucows-donejs-carousel {
 	}
 
 	.slideTrack {
-		display: flex;
+		display: grid;
```

Comment:
still need figure out how to provide option for fade vs slide transition here in the .less